docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -684,4 +684,138 @@ The default is 9000.
   -V, --version
 ``` 
 
-Displays the version and exits. 
+Displays the version and exits.
+
+### validators-external-signer-public-keys
+
+```bash tab="Syntax"
+--validators-external-signer-public-keys=<KEY>[,<KEY>...]
+```
+
+```bash tab="Command Line"
+
+--validators-external-signer-public-keys=0xa99a...e44c,0xb89b...4a0b
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS-EXTERNAL-SIGNER-PUBLIC-KEYS=0xa99a...e44c,0xb89b...4a0b
+```
+
+```bash tab="Configuration File"
+validators-external-signer-public-keys=[
+  "0xa99a...e44c",
+  "0xb89b...4a0b"]
+```
+
+List of validator public keys used by an external signer (for example Eth2Signer).
+
+### validators-external-signer-timeout
+
+```bash tab="Syntax"
+--validators-external-signer-timeout=<INTEGER>
+```
+
+```bash tab="Command Line"
+--validators-external-signer-timeout=2000
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS-EXTERNAL-SIGNER-TIMEOUT=2000
```

Comment:
Same comment as above re hyphens.  Same comment applies to all following options

Reply:
Done